sdk/eventhub/event-hubs/README.md

```diff
@@ -181,6 +182,46 @@ for await (const events of consumer.getEventIterator()){
 }
 ```
 
+### Consume events using an Event Processor
+
+To consume events for all partitions of an Event Hub, create an `EventProcessor` for a specific consumer group. When an Event Hub is created, it provides a default consumer group that can be used to get started.
+
+The `EventProcessor` will delegate the processing of events to a `PartitionProcessor` that you provide, allowing you to focus on business logic while the processor holds responsibility for managing the underlying consumer operations. In our example, we will focus on building the `EventProcessor` and use a very minimal partition processor that does no actual processing.
+
+```javascript
+class SimplePartitionProcessor {
+  async processEvents(events: ReceivedEventData[]) {
+    // your code here
+  }
+
+  async processError(error: Error) {
+    // your error handler here
+  }
+}
+
+const processor = new EventProcessor(
+  EventHubClient.defaultConsumerGroupName,
+  client,
+  () => {
+    return new SimplePartitionProcessor();
+  },
```

Comment:
Lets we collapse the above 3 lines to 1

Reply:
This is because of formatter. I'll change to 
```javascript
const processor = new EventProcessor(
  EventHubClient.defaultConsumerGroupName,
  client,
  () => new SimplePartitionProcessor(),
  new InMemoryPartitionManager()
);
```